### DISCUSSION TITLE
为什么我功能页没有下载视频这个功能，设置中已打开

### DISCUSSION THREAD

**Discussion by Continue007** [the1812/Bilibili-Evolved#3284] [提问 / Q&A] [answered]
如题及截图
![下载视频](https://user-images.githubusercontent.com/104631969/165946149-033ad35c-a0ca-4b0f-8002-dfc45b9f6349.png)
![功能页](https://user-images.githubusercontent.com/104631969/165946323-e9ce849b-b991-496d-83cb-3dccf568fc33.png)

**Comment by Geeyun-JY3** [comment, marked as answer]
看 issue 区置顶，检查播放器版本：#3187

**Reply by Continue007** [reply]
感谢
